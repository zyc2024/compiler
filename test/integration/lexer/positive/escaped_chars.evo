'\n'
'\t'
'\r'
'\x{0a}'
'\"'
'\''
'\u0040'