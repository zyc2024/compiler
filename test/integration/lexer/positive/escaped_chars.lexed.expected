1:1 character '\n'
2:1 character '\t'
3:1 character '\r'
4:1 character '\n'
5:1 character '\"'
6:1 character '''
7:1 character '@'